pennylane/devices/default_clifford.py

```diff
@@ -0,0 +1,579 @@
+            this device.
+        check_clifford (bool): Check if all the gate operations in the circuits to be executed are Clifford. Default is ``True``.
+        tableau (bool): Determines what should be returned when the device's state is computed with :func:`qml.state <pennylane.state>`.
+            which makes the device return the final evolved Tableau. Alternatively, one may make it ``False`` to obtain
+            the evolved state vector. Note that the latter might not be computationally feasible for larger qubit numbers.
+        seed (Union[str, None, int, array_like[int], SeedSequence, BitGenerator, Generator]): A
+            seed-like parameter matching that of ``seed`` for ``numpy.random.default_rng``, or
+            a request to seed from numpy's global random number generator.
+            The default, ``seed="global"`` pulls a seed from NumPy's global generator. ``seed=None``
+            will pull a seed from the OS entropy.
+        max_workers (int): A ``ProcessPoolExecutor`` executes tapes asynchronously
+            using a pool of at most ``max_workers`` processes. If ``max_workers`` is ``None``,
+            only the current process executes tapes. If you experience any
+            issue, try setting ``max_workers`` to ``None``.
+
+    **Example:**
+
+    The :class:`~pennylane.devices.DefaultClifford` implements the ``default.clifford`` device,
+    which can be used for efficiently executing circuits built with
+    `Clifford gates <https://en.wikipedia.org/wiki/Clifford_gates>`_.
+
+    .. code-block:: python
+
+        dev = qml.device("default.clifford", tableau=True)
+
+        @qml.qnode(dev)
+        def circuit():
+            qml.CNOT(wires=[0, 1])
+            qml.PauliX(wires=[1])
+            qml.ISWAP(wires=[0, 1])
+            qml.Hadamard(wires=[0])
+            return qml.state()
+
+    >>> circuit()
+    array([[0, 1, 1, 0, 0],
+            [1, 0, 1, 1, 1],
+            [0, 0, 0, 1, 0],
+            [1, 0, 0, 1, 1]])
+
+    A more fine-grained control over the execution pipeline can be obtained with
+    :class:`~pennylane.tape.QuantumScript`.
+
+    .. code-block:: python
+
+        num_qscripts = 5
+
+        qscripts = [
+            qml.tape.QuantumScript(
+                [qml.Hadamard(wires=[0]), qml.CNOT(wires=[0, 1])],
+                [qml.expval(qml.PauliZ(0))]
+            )
+        ] * num_qscripts
+
+    >>> dev = DefaultClifford()
+    >>> program, execution_config = dev.preprocess()
+    >>> new_batch, post_processing_fn = program(qscripts)
+    >>> results = dev.execute(new_batch, execution_config=execution_config)
+    >>> post_processing_fn(results)
+    (array(0), array(0), array(0), array(0), array(0))
+
+    .. details::
+        :title: Clifford Tableau
+        :href: clifford-tableau-theory
+
+        The device represents a state by the inverse of a Tableau
```

Comment:
In Stim, instead of storing the tableau, the inverse of the Tableau (T) is stored instead of the Tableau itself (T^-1).  So to get the state, we manually invert it when a user would call `qml.state()`.

Reply:
Removed the term "inverse" from here.